docs/eventlisteners.md

```diff
@@ -71,11 +71,14 @@ See our [Tekton Triggers examples](https://github.com/tektoncd/triggers/tree/mas
 ## Specifying the Kubernetes service account
 
 You must specify a Kubernetes service account in the `serviceAccountName` field that the `EventListener` will use to instantiate Tekton objects.
-This account must have the following assigned:
-- A Kubernetes `Role` that permits the `get`, `list`, and `watch` verbs for each `Trigger` specified in the `EventListener`
-- A Kubernetes `ClusterRole` that permits read access to `ClusterTriggerBindings` objects
-- Permissions to create the Tekton resources specified in the associated `TriggerTemplate`, as shown in the following [example](../examples/rbac.yaml)
-- If you're using `namespaceSelectors` in your `EventListener`, a `ClusterRole` that permits read access to all `Trigger` objects on the cluster.ources across the cluster.
+
+Tekton Trigger creates 2 clusterroles while installing with necessary permissions required for an eventlistener. You can directly create bindings for your serviceaccount with the clusterroles.
+- A Kubernetes RoleBinding with `tekton-triggers-eventlistener-roles` clusterrole.
```

Comment:
Issue with this is that it allows us for [impersonation ](https://github.com/tektoncd/triggers/commit/4056b0374510b93cf388b4fe71afb173170a1510#diff-d5a724e46fb38be29b6dc700b6799f79f9d9178c6729ef5172d477f022918bb2R75) clusterwide.
@dibyom Is this OK? Earlier we were using role not clusterrole so it wasn't a issue.

Reply:
We are using a RoleBinding to use this ClusterRole, so the impersonation should still be gated to the namespace...right?

Reply:
Discussed at WG today -- using a RoleBinding should be good enough!
@khrm  let me know if you have any concerns!